core/src/main/java/com/datastrato/gravitino/storage/kv/KvEntityStore.java

```diff
@@ -323,15 +324,19 @@ private List<byte[]> getSubEntitiesPrefix(NameIdentifier ident, EntityType type)
         prefixes.add(replacePrefixTypeInfo(encode, CATALOG.getShortName()));
         prefixes.add(replacePrefixTypeInfo(encode, SCHEMA.getShortName()));
         prefixes.add(replacePrefixTypeInfo(encode, TABLE.getShortName()));
+        prefixes.add(replacePrefixTypeInfo(encode, FILESET.getShortName()));
         break;
       case CATALOG:
         prefixes.add(replacePrefixTypeInfo(encode, SCHEMA.getShortName()));
         prefixes.add(replacePrefixTypeInfo(encode, TABLE.getShortName()));
+        prefixes.add(replacePrefixTypeInfo(encode, FILESET.getShortName()));
         break;
       case SCHEMA:
         prefixes.add(replacePrefixTypeInfo(encode, TABLE.getShortName()));
+        prefixes.add(replacePrefixTypeInfo(encode, FILESET.getShortName()));
         break;
       case TABLE:
+      case FILESET:
```

Comment:
Also here, I think it is necessary to have UTs to cover it.

Reply:
@yuqi1129 can you please help to review this part and the UT added below?

Reply:
@jerryshao 
He has added test cases about it, please see `validateDeleteSchemaCascade`, `validateDeleteCatalogCascade`, and `validateDeleteFilesetCascade`

Reply:
The other parts are fine for me.